package.json

```diff
@@ -10,34 +10,39 @@
     "startd": "echo 'Do you mean 'npm run watch'?'",
     "test": "mocha 'test/**/*.js' --recursive --timeout 60000 --exit"
   },
+  "engines": {
+    "node": "16",
```

Comment:
You have to run npm install again, because the engines are not yet in package-lock.json.